src/cli/report.rs

```diff
@@ -203,6 +208,8 @@ impl Report for BgpAnalysisSuggestion {}
 
 impl Report for AspaDefinitionList {}
 
+impl Report for BgpSecCsrInfoList {}
```

Comment:
In a nutshell because `BgpSecDefinition` contains a validly parsed BGPSec certificate sign request, whereas the `Info` type is what we get from the saved definitions. We may not consider them all valid in future. Especially since this is our first (beta) implementation of all this.